app/views/subscriptions/check_email.html.erb

```diff
@@ -1,20 +1,29 @@
-<% content_for :back_link do %>
-  <%= render "govuk_publishing_components/components/back_link", {
-      href: @back_url
-  } %>
-<% end %>
-
 <% content_for :title, t("subscriptions.check_email.title") %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= tag.h1 t("subscriptions.check_email.title"), class: "govuk-heading-l" %>
+    <h1 class="govuk-heading-l"><%= t("subscriptions.check_email.title") %></h1>
+
+    <p class="govuk-body">
+      <%= t("subscriptions.check_email.description_html", address: @address) %>
+    </p>
+
+    <p class="govuk-body">
+      <%= t("subscriptions.check_email.instruction") %>
+    </p>
+
+    <p class="govuk-body">
+      <%= tag.strong @title %>
```

Comment:
Did you mean this to be `<strong><%= @title %></strong>` ?

Reply:
Yup - updated 👍